### PR TITLE
large numbers converted to k notation

### PR DESCRIPTION
**Pull request by Gamsti1337**
**Please describe the changes this PR makes and why it should be merged, as well as any and all proof of successful tests:**

**Semantic versioning classification:**

- [x] This PR changes the framework's interface (methods or parameters added)
  - [ ] This PR includes breaking changes (methods removed or renamed, parameters moved or removed)
- [ ] This PR **only** includes non-code changes, like changes to README, etc.

In this PR I coded a function to change large numbers to a K notation (69420 = 69.4k) and removed duplicate stats on /bot command (like servers and total servers) #103

Some changes influenced by discord are there like:
> putting the deploy command functionality in ready.js. Why? Because when trying to setup I needed to /deploy to show all slash commands... but I didn't have any slash commands to use /deploy
> checking if some value is undefined in a file I forgot

### CODE REVIEW

**Review by OmerSabic — APPROVED** [review]
lgtm

**Comment by Gamsti1337** [comment]
@OmerSabic can you merge?

**Comment by OmerSabic** [comment]
> @OmerSabic can you merge?

I don't have the permissions to

**Comment by Gamsti1337** [comment]
lets wait for sean

**Comment by Gamsti1337** [comment]
@Project-Skill-Tree/management can you merge?

File: .env.example [hunk 1]
```diff
@@ -1,4 +0,0 @@
-DISCORD_TOKEN=
-OWNER=
-API_URL=
```
Comment (swatarianess) [inline .env.example:3]: Don't delete this file, its needed 😉

**Review by swatarianess — CHANGES_REQUESTED** [review]

**Review by seanjhardy — CHANGES_REQUESTED** [review]
Fix the units of number formatter, and make sure it actually returns a value for every input (will fail for 900-999)

**Comment by seanjhardy** [comment]
Smh omer look more thoroughly

**Comment by Gamsti1337** [comment]
rewrote the function, can you check now?

**Review by seanjhardy — APPROVED** [review]

**Comment by OmerSabic** [comment]
> Smh omer look more thoroughly

yep my bad, gonna check better from now on

**Review by swatarianess — APPROVED** [review]

**Pull request merged**